pulsar-common/pom.xml

```diff
@@ -166,6 +166,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Is this needed?

Reply:
Ya you probably don't need to import Gson.  Just use Jackson i.e. `ThreadObjectFactory.getThreadLocal()`